src/databricks/labs/remorph/config.py

```diff
@@ -104,6 +104,13 @@ class ReconcileTablesConfig:
     tables_list: list[str]  # [*, table1, table2]
 
 
+@dataclass
+class ReconMetricsConfig:
+    catalog: str
+    schema: str
+    volume: str = "RECONCILE_VOLUME"
```

Comment:
'RECONCILE_VOLUME' should be in lower case 'reconcile_volume'
Since you are using 'reconcile_volume' we should change the fixed variable in 
reconcile/execute.py _RECONCILE_VOLUME variable FYI